lambda-http4s/src/main/scala/feral/lambda/http4s/ApiGatewayProxyHandler.scala

```diff
@@ -114,6 +95,15 @@ object ApiGatewayProxyHandler {
         .mkString("&")
     }
 
+  private def getMultiValueHeaders(
+      multiValueHeaders: Option[Map[CIString, List[String]]]): List[Header.Raw] = {
+    for {
+      headerMap <- multiValueHeaders.toList
+      (key, values) <- headerMap
+      headerRaw <- values.map(value => Header.Raw(key, value))
+    } yield headerRaw
+  }
```

Comment:
This implementation uses a lot of intermediate allocations, would you mind re-using the original `ListBuilder`-based implementation?